web/src/components/profileEdit/QuotaCardEdit.tsx

```diff
@@ -128,8 +147,28 @@ const QuotaCardEdit: React.FC<IQuotaCardEditProps> = (props) => {
         {specs.storageNums[1]} overall storage with {specs.storageNums[2]} for backup storage
       </Text>
       <br />
+
+      <p>
```

Comment:
During review this portion of code seems to cause an issue with the layout, as seen in the attached screen shot. The expected result would be for the Quota information card to populate ~60% of the window with the text blurb on the right side as seen in the second screenshot.

Additionally, the use of flat HTML has typically avoided. You will see in this page an example on line #144 that demonstrates the use of <Text>. I will note, this page could definitely be subject to a more thorough UI/UX enhancement if you wished to rewrite the component to be more user friendly. I can draft a ticket if that were the case.
```suggestion
      <Text as='p'>
```


<img width="1653" alt="Screen Shot 2021-07-12 at 4 27 31 PM" src="https://user-images.githubusercontent.com/30994898/125367950-124d7900-e32e-11eb-8b01-213061ec595c.png">
<img width="1653" alt="Screen Shot 2021-07-12 at 4 29 30 PM" src="https://user-images.githubusercontent.com/30994898/125368155-7ff9a500-e32e-11eb-8249-f0c5893281db.png">

Reply:
I have updated using styled component and add a style to it.